recipes/wdiff/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "wdiff" %}
+{% set version = "1.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://ftp.gnu.org/gnu/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 34ff698c870c87e6e47a838eeaaae729fa73349139fc8db12211d2a22b78af6b
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make
+    - texinfo
+  host:
+    - ncurses  # [linux]
+  run:
+    - ncurses  # [linux]
```

Comment:
Isn't this needed on macOS as well?

Reply:
Does the `run` requirement need `{{ pin_compatible(...) }}` instead?

Reply:
I don't know enough about conda.

Vim does it exactly like this with ncurses.

Reply:
No, you should be able to omit the run requirements as `ncurses` has a `run_exports` and adds the correct `pin_compatible` automatically on build.

Reply:
@xhochy, the `run_export` was added with a build number bump though as opposed to a new version ( https://github.com/conda-forge/ncurses-feedstock/pull/58/commits/4858e6927a11cc51644b06c653f67f88ae8a35ff ), which means if an older build number is picked up we could be missing `run_exports`

Reply:
The recipe wouldn't pass the tests if the non-run_exports version is used and `ncurses` isn't listed in `run`. So not listing it there actually ensures that we have the correct pinning. This I would certainly remove it from `run`

Reply:
Ah ok. Fair enough